sample-apps/payment-customization-sample-app/extensions/hide-payment-by-name-and-cart-subtotal/src/main.rs

```diff
@@ -0,0 +1,167 @@
+use serde::{Deserialize, Serialize};
+
+mod api;
+use api::*;
+
+#[derive(Clone, Debug, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct Configuration {
+    pub payment_method: String,
+    pub cart_subtotal: u64,
```

Comment:
@DuncanUszkay1 Can/should we make some helper module to deal with `Money` objects with `subunits` and `currency`? It's pretty confusing to convert these values into presentment currency

Reply:
In Rust?

Reply:
Maybe having a better tutorial around how to deal with the merchant-configured Money value would be helpful 😅

It's confusing and not obvious how to compare input Money objects that have subunits in the presentment currency to some arbitrary value in your app UI.

Reply:
Created an issue to make that improvement, we can triage it next week at planning :+1: